catalog/to-consul/syncer.go

```diff
@@ -218,18 +243,26 @@ func (s *ConsulSyncer) watchService(ctx context.Context, name string) {
 		case <-time.After(s.SyncPeriod):
 		}
 
+		// Set up query options
+		queryOpts := &api.QueryOptions{
+			AllowStale: true,
+		}
+		if s.EnableNamespaces {
+			// Sets the Consul namespace to query the catalog
+			queryOpts.Namespace = namespace
```

Comment:
again, what happens if this is set to "default" when there are no namespaces enabled?

Reply:
Answered above, the call fails.